### PR TITLE
Version bump for 2.10.0

### PR DESCRIPTION
**Pull request by vitaliyreznikov**
# Description
Please include a summary of the change and which issue is fixed. Include the motivation for the changes, and comment on your PR if necessary for clarity.

Fixes: (link Jira Task)

#changelog Version bump for 2.10.0

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Test

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [ ] Successfully tested locally (or docker image)
- [ ] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have created or modified unit tests to sufficiently cover my changes.
- [ ] I have added my Asana task link and provided a changelog message if applicable.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # Changelog
+## [V2.10.0](https://bolt-public.s3-us-west-1.amazonaws.com/magento-integration-release/bolt-magento1-v2.10.0.zip) 2020-11-05
+ - added: custom server URLs for staging and dev are now supported.
```
Comment (adentong) [inline CHANGELOG.md:3]:
```suggestion
 - Added support for custom URLs for staging and dev
```

**Review by adentong — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (vitaliyreznikov) [reply CHANGELOG.md]: @adentong This is a new approach we already use in M2 for a few months
https://github.com/BoltApp/bolt-magento2/blob/master/CHANGELOG.md
CC: @lbliii

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
 # Changelog
+## [V2.10.0](https://bolt-public.s3-us-west-1.amazonaws.com/magento-integration-release/bolt-magento1-v2.10.0.zip) 2020-11-05
+ - Added support for custom URLs for staging and dev
```
Comment (adentong) [inline CHANGELOG.md:3]:
```suggestion
 - Added: custom URLs for staging and dev are not supported.
```

File: CHANGELOG.md (hunk 1 above)
Comment (adentong) [reply CHANGELOG.md]: @vitaliyreznikov updated

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,6 @@
 # Changelog
+## [V2.10.0](https://bolt-public.s3-us-west-1.amazonaws.com/magento-integration-release/bolt-magento1-v2.10.0.zip) 2020-11-05
+ - Added: custom URLs for staging and dev are not supported.
```
Comment (adentong) [inline CHANGELOG.md:3]:
```suggestion
 - Added: custom URLs for staging and dev are now supported.
```

**Review by johnmccombs1 — APPROVED** [review]

**Review by adentong — APPROVED** [review]

**Pull request merged**